schema/defs.yaml

```diff
@@ -343,6 +343,16 @@ description: Common schema definitions shared by all themes
           title: cartography
           type: object
           properties:
+            prominence:
+              description: >-
+                Represents Overture's view of a place's significance or importance.
+                This value can be used to help drive cartographic display of an
+                administrative place and is derived from various factors including:
```

Comment:
`s/an administrative place/a feature` ?

Reply:
This should be resolved - I've updated this description to be "Represents Overture's view of a place's significance or importance. This value can be used to help drive cartographic display of a place and is derived from various factors including, but not limited to: population, capital status, place tags, and type."